src/script/standard.cpp

```diff
@@ -98,6 +98,16 @@ bool Solver(const CScript& scriptPubKey, txnouttype& typeRet, vector<vector<unsi
         return true;
     }
 
+    // Zerocoin to Sigma remint
+    if (scriptPubKey.IsZerocoinRemint())
+    {
+        typeRet = TX_ZEROCOINTOSIGMAREMINT;
```

Comment:
please add scriptPubkey size assertion.

Reply:
Sure

Reply:
Wait!, sorry I just notice that. As I understand function `Solver` is use only with output.

Reply:
From it's signature, it can be used with any CScript object. There is nothing wrong in using it with inputs. Actually, the code here is not used as I found a shorter way of handing this. Nonetheless, this code is useful for debugging.

Reply:
The first parameter name and type of the second parameter which is `enum txnouttype` indicate clearly what kind of script need to be handled by the function. Also I don't think debugging purpose is good enough reason for add unused code.

Reply:
Ok, removed

Reply:
Here you are

Reply:
LGTM. thanks for removing.